src/api/app/controllers/status/reports_controller.rb

```diff
@@ -0,0 +1,96 @@
+class Status::ReportsController < ApplicationController
```

Comment:
Do we need this controller? :thinking:

Reply:
I'm fine with keeping the index but:

* show is basically the checks index
* create is not needed as a report gets implicit created when creating a new check - see checks controller
* update and destroy should also not be possible as the report gets implicit created so I don't expect that it is possible to get updated